examples/notepad/src/main/kotlin/util/AwtIcons.kt

```diff
@@ -0,0 +1,77 @@
+package util
+
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.geometry.isSpecified
+import androidx.compose.ui.graphics.Canvas
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.graphics.ImageBitmap
+import androidx.compose.ui.graphics.drawscope.CanvasDrawScope
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.graphics.vector.rememberVectorPainter
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.platform.LocalLayoutDirection
+import androidx.compose.ui.unit.Density
+import androidx.compose.ui.unit.LayoutDirection
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import java.awt.Point
+import java.awt.image.BufferedImage
+import java.awt.image.ColorModel
+import java.awt.image.DataBuffer
+import java.awt.image.DataBufferInt
+import java.awt.image.Raster
+import java.awt.image.SinglePixelPackedSampleModel
+import kotlin.math.roundToInt
+
+suspend fun ImageVector.toAwtImage(tintColor: Color): BufferedImage {
```

Comment:
Maybe add those(non-suspend versions) to https://github.com/JetBrains/skiko/blob/master/skiko/src/jvmMain/kotlin/org/jetbrains/skiko/Convertors.kt

Reply:
ImageVector is a Compose class, so we can add it to Compose in the future

Reply:
We can replace part of this code by this:
```
    CanvasDrawScope().draw(
        density, layoutDirection, canvas, intrinsicSize
    ) {
        draw(intrinsicSize, colorFilter = colorFilter)
    }

    return bitmap.asDesktopBitmap().toBufferedImage()
```

Icon in Tray is ok, but icon in titlebar is not ok. Something is wrong in `toBufferedImage` (will figure out this later).